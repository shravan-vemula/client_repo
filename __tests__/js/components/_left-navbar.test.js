import React from 'react';
import {render,cleanup} from '@testing-library/react';
import LeftNavBar from 'js/_components/organisms/_left-navbar';
import{ BrowserRouter } from 'react-router-dom';

afterEach(cleanup)

describe("LeftNavBar Component",()=>{
    it("renders correctly", () => {
        const wrapper = shallow(<LeftNavBar />);
        expect(wrapper).toMatchSnapshot();
      });

    it("should have all the icons", ()=>{
        const{ getAllByRole } = render(<BrowserRouter><LeftNavBar /></BrowserRouter>);
        expect(getAllByRole("img")).toBe;
    });

});