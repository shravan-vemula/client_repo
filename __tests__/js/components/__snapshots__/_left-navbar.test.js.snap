// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftNavBar Component renders correctly 1`] = `ShallowWrapper {}`;
